src/data/modifiers.txt

```diff
@@ -8266,15 +8321,16 @@ Skill	Limberger Limberness	Initiative: +75
 Skill	Liver of Steel	Liver Capacity: +5
 Skill	Localized Vacuum	Hot Resistance: +2
 Skill	Long Winter's Nap	Free Rests: 5
-# Louder Bellows: You can bellow louder
 # Louder Bellows: (bellow delevels by 30%, and also stuns the monster for one round)
+# Louder Bellows: You can bellow louder
 Skill	Lower Hypothalamus	Cold Resistance: +1
 Skill	Lubricant Layer	Sleaze Resistance: +2
 Skill	Lucky Brooch	Item Drop: +30
 # Lucky Buckle: Find more coins after combat
 Skill	Lucky Insignia	Item Drop: +30
 Skill	Lucky Pin	Item Drop: +30
 Skill	Lunch Like a King	Stomach Capacity: +5
+Skill	Lust	Initiative: +50, Weapon Damage: -5, Spell Damage: -5
```

Comment:
a single comment for all bad moon skills: the bad moon skills were formerly separate.